plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHiveOnDataLake.java

```diff
@@ -2095,6 +2095,27 @@ public void testDropStatsPartitionedTable()
         assertUpdate("DROP TABLE " + getFullyQualifiedTestTableName(tableName));
     }
 
+    @Test
+    public void testUnsupportedDropSchemaCascadeWithNonHiveTable()
+    {
+        String schemaName = "test_unsupported_drop_schema_cascade_" + randomNameSuffix();
```

Comment:
Can we add a hive tables to ensure it is also no dropped ?

Reply:
The ordering when dropping objects isn't ensured.